packages/manager/modules/adp/src/deploy/deploy.scss

```diff
@@ -0,0 +1,52 @@
+// override bootstrap map to add custom widths
+// This variable affects the `.h-*` and `.w-*` classes.
+$sizes: () !default;
+// stylelint-disable-next-line scss/dollar-variable-default
+$sizes: map-merge(
+  (
+    10: 10%,
+    15: 15%,
+    20: 20%
+  ),
+  $sizes
+);
+
+@import "~bootstrap4/scss/_functions";
```

Comment:
There are mixed `'` versus `"` and some extra `.scss` in some imports statement.

Reply:
This is fixed